modules/core/message_functions.php

```diff
@@ -16,16 +16,26 @@
 if (!hm_exists('format_msg_html')) {
 function format_msg_html($str, $images=false) {
     $str = str_ireplace('</body>', '', $str);
+
     $config = HTMLPurifier_Config::createDefault();
+    $config->set('HTML.DefinitionID', 'hm-message');
+    $config->set('HTML.DefinitionRev', 1);
     $config->set('Cache.DefinitionImpl', null);
     $config->set('HTML.TargetBlank', true);
     $config->set('HTML.TargetNoopener', true);
-    $config->set('HTML.Allowed', 'a[href|target]');
```

Comment:
Don't we want to keep links in html content? We even had recently added external links to open in new tabs. Wouldn't this change remove the links from the purified content?

Reply:
I believe not. Links are present by default. This configuration allows only the `a` tag and does not return the entire message content when I inspected it.

Reply:
Alright, I am not very familiar with the purifier.